.github/workflows/.test.yml

```diff
@@ -396,9 +403,9 @@ jobs:
       -
         name: Upload reports
         if: always()
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
-          name: ${{ inputs.storage }}-integration-cli-reports
+          name: test-reports-integration-cli-${{ inputs.storage }}-${{ env.TESTREPORTS_NAME }}
```

Comment:
Is this because it wants / needs to group by prefix?

Reply:
Yes that's it as v4 requires a bundle per upload so we can download by pattern later on.